libs/solid/profile/src/lib/services/profile-definition.service.ts

```diff
@@ -92,7 +92,7 @@ export class ProfileDefinitionService {
     const { title, type } = schema;
     const required = parent.required?.includes(key) ?? false;
     // tslint:disable:no-non-null-assertion
-    switch (type) {
+    switch (type as any) {
```

Comment:
Das verstehe ich zwar nicht ganz, aber es wird einen Grund haben 😉

Reply:
Ohne _any_ die neuen Definitionen von _mdstring_ und _colstring_ gingen nicht. Ich verstehe auch nicht ganz aber die ist die einzige Lösung die ich gefunden habe um es zu lösen.

Reply:
Hm... eigentlich sollte es nicht nötig sein, gerade in Typescript sollten wir ja wissen, welche Datentypen wir kriegen.
Es kann sein, dass es irgendwo bei Swagger noch definiert werden? Vielleicht hat @mkay-sh eine Idee dazu.

Reply:
Laut OpenAPI gibt es den PropertyType colstring bzw. mdstring nicht. Deswegen muss man da casten. Allerdings wäre es sicherer in einen UnionType zu casten. Any ist immer fehleranfällig.
Vorschlag von mir wäre 'type as UrsprünglicherType | "colstring" | "mdstring"'
Dann bleibt die Typisierung erhalten.